### PR TITLE
Added generate.ps1, removed generate.cmd, generated files.

### PR DESCRIPTION
**Pull request by sriramvu**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/_metadata/recoveryservicessiterecovery_resource-manager.txt [hunk 1]
```diff
@@ -1,17 +1 @@
-﻿Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/recoveryservicessiterecovery/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=D:\Github\azure-sdk-for-net\tools\..\src\SDKs\
-2018-05-08 13:44:39 UTC
-1) azure-rest-api-specs repository information
-GitHub fork: Azure
-Branch:      master
-Commit:      f0face31d69f4a7191f9db6143c0a1a01f398d89
```
Comment (dsgouda) [inline src/SDKs/_metadata/recoveryservicessiterecovery_resource-manager.txt]: @sriramvu Please run msbuild build.proj and regenerate the code again

File: src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery.csproj [hunk 2]
```diff
@@ -6,14 +6,15 @@
...
+		<Version>1.2.1-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.RecoveryServices.SiteRecovery</AssemblyName>
 		<PackageTags>Recoveryservices;SiteRecovery;</PackageTags>
 		<PackageReleaseNotes>
 			<![CDATA[
-        Updates in this release:
-			- Fixes corresponding to ARM compliance in swagger introducing new API version (2018-01-10) and taking other bugfixes like representing enums as strings.
-      ]]>
+		        Updates in this release:
+			- Introduced generate.ps1 & generated files out of latest published swagger json.
```
Comment (dsgouda) [inline src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery.csproj]: This is repo-specific, package release notes are viewed on nuget and should only be pertinent to the SDK itself

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks great for the most part. left a couple of comments

File: src/SDKs/_metadata/recoveryservicessiterecovery_resource-manager.txt (hunk 1 above)
Comment (sriramvu) [reply src/SDKs/_metadata/recoveryservicessiterecovery_resource-manager.txt]: regenerated code and SdkInfo_SiteRecoveryManagementClient.cs got some changes.

File: src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery.csproj (hunk 2 above)
Comment (sriramvu) [reply src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery.csproj]: Corrected the release notes to have only SDK related changes.

**Review by dsgouda — APPROVED** [review]
LGTM subject to CIs passing

**Pull request merged**